### PR TITLE
Update MXE build recipes

### PR DESCRIPTION
**Pull request by elpaso**
This was required by upstream updates to newer QT and CMake

### CODE REVIEW

File: ms-windows/mxe/build-mxe.sh [hunk 1]
```diff
@@ -24,6 +24,8 @@ set -e
 #        The artifact will be saved as a zip package in the directory
 #        from which this script is launched.
 
+# shellcheck disable=SC2155
```
Comment (nyalldawson) [inline ms-windows/mxe/build-mxe.sh]: Why not fix this instead? https://github.com/koalaman/shellcheck/wiki/SC2155

Comment (elpaso) [reply ms-windows/mxe/build-mxe.sh]: Because it's nonsense, pwd in that case will never return false

Comment (elpaso) [reply ms-windows/mxe/build-mxe.sh]: But I guess I have no choice than wasting my time :(

Comment (jef-n) [reply ms-windows/mxe/build-mxe.sh]: $PWD?

**Comment by elpaso** [comment]
Superceeded by #33064

**Pull request closed without merging**